packages/fiori/src/ViewSettingsDialog.hbs

```diff
@@ -1,48 +1,57 @@
 <ui5-dialog
+	dir="{{effectiveDir}}"
 	aria-label="{{_dialogTitle}}"
 	@ui5-after-open={{_focusRecentlyUsedControl}}
 	@ui5-before-close={{_restoreConfirmedOnEscape}}
 	?stretch={{_isPhone}}
 >
 
 	<div slot="header" class="ui5-vsd-header">
-		<ui5-bar>
-			<div slot="startContent" class="ui5-vsd-start">
+		<div class="ui5-vsd-header-container">
+			<ui5-bar class="ui5-vsd-bar-header">
 				{{#if showBackButton}}
 					<ui5-button
+						slot="startContent"
 						design="Transparent"
 						icon="nav-back"
 						class="ui5-vsd-back-button"
 						@click="{{_navigateToFilters}}"
 					></ui5-button>
-					<ui5-label>{{_filterByTitle}}</ui5-label>
+					<ui5-label slot="startContent" class="ui5-vsd-title">{{_filterByTitle}}</ui5-label>
```

Comment:
You can move this out of the if statement. This way you won't need to have else statement

Reply:
done